app/components/windows/go-live/platforms/FacebookEditStreamInfo.tsx

```diff
@@ -273,18 +275,24 @@ export default class FacebookEditStreamInfo extends BaseEditSteamInfo<Props> {
     electron.remote.shell.openExternal(`https://www.facebook.com/groups/${groupId}/edit`);
   }
 
+  private dismissWarning() {
+    this.dismissablesService.actions.dismiss(EDismissable.FacebookNeedPermissionsTip);
+  }
+
   render() {
     const hasPages = this.facebookService.state.facebookPages.length;
     const fbSettings = this.settings.platforms.facebook;
     const shouldShowGroups = fbSettings.destinationType === 'group' && !this.props.isUpdateMode;
     const shouldShowPages = fbSettings.destinationType === 'page' && !this.props.isUpdateMode;
-    const shouldShowPermissionWarn = !this.canStreamToTimeline || !this.canStreamToGroup;
     const shouldShowGamingWarning = !hasPages && fbSettings.game;
     const shouldShowEvents = !this.props.isUpdateMode && !this.props.isScheduleMode;
     const shouldShowPrivacy = fbSettings.destinationType === 'me';
     const shouldShowPrivacyWarn =
       (!fbSettings.liveVideoId && fbSettings.privacy.value !== 'SELF') ||
       (fbSettings.liveVideoId && fbSettings.privacy.value);
+    const shouldShowPermissionWarn =
+      (!this.canStreamToTimeline || !this.canStreamToGroup) &&
+      !this.dismissablesService.state[EDismissable.FacebookNeedPermissionsTip];
```

Comment:
Not a big deal, but there's also a view for this: `this.dismissablesService.views.shouldShow(EDmissable.FacebookNeedPermissionsTip)`